app/src/main/res/layout/activity_settings.xml

```diff
@@ -8,79 +8,103 @@
-        android:layout_width="wrap_content"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginStart="8dp"
         android:layout_marginTop="16dp"
         android:layout_marginEnd="8dp"
-        android:padding="8dp"
+        android:padding="16dp"
         android:text="@string/popup_menu_feedback"
         android:textColor="@android:color/black"
         android:textSize="18sp"
+        android:gravity="center_vertical"
+        android:drawableLeft="@drawable/ic_feedback_black_24dp"
+        android:drawablePadding="8dp"
+        android:background="?attr/selectableItemBackground"
+        android:clickable="true"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintHorizontal_bias="0.02"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/tvAbout" />
 
     <androidx.appcompat.widget.AppCompatTextView
         android:id="@+id/tvLogout"
-        android:layout_width="wrap_content"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginStart="8dp"
         android:layout_marginTop="16dp"
         android:layout_marginEnd="8dp"
-        android:padding="8dp"
+        android:padding="16dp"
         android:text="@string/popup_menu_logout"
         android:textColor="@android:color/black"
         android:textSize="18sp"
+        android:gravity="center_vertical"
+        android:drawableLeft="@drawable/ic_exit_to_app_black_24dp"
+        android:drawablePadding="8dp"
+        android:background="?attr/selectableItemBackground"
+        android:clickable="true"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintHorizontal_bias="0.02"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/tvFeedback" />
 
     <androidx.appcompat.widget.AppCompatTextView
         android:id="@+id/tvResetPassword"
-        android:layout_width="wrap_content"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginStart="8dp"
         android:layout_marginTop="16dp"
         android:layout_marginEnd="8dp"
-        android:padding="8dp"
+        android:padding="16dp"
         android:text="@string/change_password"
         android:textColor="@android:color/black"
         android:textSize="18sp"
+        android:gravity="center_vertical"
+        android:drawableLeft="@drawable/ic_lock_black_24dp"
+        android:drawablePadding="8dp"
+        android:background="?attr/selectableItemBackground"
+        android:clickable="true"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintHorizontal_bias="0.02"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/tvLogout" />
 
     <androidx.appcompat.widget.AppCompatTextView
```

Comment:
this button is missing the clickable effect right? @jenpaff

Reply:
Yes the delete button was missing the effect, I just added it. I'm really sorry, I'm going to be more careful when testing next time!

Reply:
No need to be sorry ;) I review to catch these types of things, no worries ;) @jenpaff